### DISCUSSION TITLE
Learn system of ODE

### DISCUSSION THREAD

**Discussion by edoardo100** [lululxvi/deepxde#845] [Q&A]
Dear Dr. Lu
I am using DeepONet in order to train a system of ODE. Is there a way to output two equations?. At the moment I can only train a single equation. I am using aligned dataset. 
Current approach for single equation is

```
# ...
data = dde.data.TripleCartesianProd(X_train=X_train, y_train=y_train, X_test=X_test, y_test=y_test)
dim_x = 1
net = dde.nn.DeepONetCartesianProd(
    [m, 40, 40],
    [dim_x, 40, 40],
    "relu",
    "Glorot normal",
)
# ...
```

`y_train` shape is `(num_sol,m)`. 
How should I generalize to  a system of two equations? Or, more in general, to n equation?
Thank you in advance

**Comment by lululxvi** [comment]
Use two DeepONets, or see https://doi.org/10.1016/j.cma.2022.114778
